src/vos/vos_aggregate.c

```diff
@@ -1192,6 +1237,47 @@ fill_segments(daos_handle_t ih, struct agg_merge_window *mw,
 	return rc;
 }
 
+static int
+process_removals(struct agg_merge_window *mw, struct vos_obj_iter *oiter, d_list_t *head, bool last,
+		 bool top)
+{
+	struct agg_rmv_ent	*rm_ent, *rm_tmp;
+	struct evt_rect		 rect;
+	int			 rc;
+
+	d_list_for_each_entry_safe(rm_ent, rm_tmp, head, re_link) {
+		rect = rm_ent->re_rect;
+
+		if (!last && rect.rc_ex.ex_hi > mw->mw_ext.ex_hi)
+			continue;
+
+		if (d_list_empty(&rm_ent->re_contained)) {
+			D_DEBUG(DB_IO, "Removing physical removal record: "DF_RECT"\n",
```

Comment:
All other debug messages in aggregate code are DB_EPC, I think we'd keep using DB_EPC.